plugins/PrivacyManager/templates/gdprOverview.twig

```diff
@@ -0,0 +1,49 @@
+{% extends 'admin.twig' %}
+
+{% set title %}{{ 'PrivacyManager_GDPR'|translate }}{% endset %}
+
+{% block content %}
+    <div piwik-content-block content-title="GDPR Overview" class="gdprOverview">
+        <p>
+            The General Data Protection Regulation (GDPR) is a regulation which strengthen and unify data protection for all individuals
+            within the European Union (EU). It allows EU citizens to better control their personal data. GDPR modernises and unifies rules allowing businesses to reduce red tape and to benefit from greater consumer trust.
+        </p>
+        <h3>Who is concerned by GDPR?</h3>
+        <p>Almost all structures dealing with EU citizens data wherever they are located are concerned by GDPR.
+            The few exemptions are listed under <a href="https://eur-lex.europa.eu/eli/reg/2016/679/oj" target="_blank" rel="noreferrer noopener">article 2 of the GDPR</a>.</p>
+        <h3>What is a personal data under GDPR?</h3>
+        <p>Any information relating to an identified or identifiable natural person. An identifiable natural person
+            is one who can be identified, directly or indirectly, in particular by reference to an identifier such as a name,
+            an identification number, location data, an online identifier...</p>
+        <h3>Where can I get more information about GDPR?</h3>
+        <p>The official GDPR text is available at <a href="https://eur-lex.europa.eu/eli/reg/2016/679/oj" target="_blank" rel="noreferrer noopener">https://eur-lex.europa.eu/eli/reg/2016/679/oj</a>. As it is interpretation may be difficult,
+            we strongly advise you to refer to your local privacy commissioner.</p>
+        <h3>How can I be GDPR compliant with Matomo Analytics?</h3>
+        <p>By default, Matomo is GDPR compliant as it is not collecting any personal data.
+<br /><br />
+            However, by changing the default settings and/or adding specific tracking code, you may collect personal data, for example:
+            IP addresses, E-commerce order ID, User ID, custom dimensions... You might be also tracking personal data for example as part of URLs or page/screen titles depending on your website or mobile app.
+<br /><br />
+            If you are processing EU citizens personal data through Matomo, you will need to fulfill the obligations listed below:
+
+        </p><ul>
+            <li>Document your use of Matomo within your information asset register.</li>
+            <li>Make sure that you are inserting our opt-out feature on all pages collecting personal data.</li>
+            <li>Is your business currently registered to an Privacy Commissioner Office?</li>
+            <li>Did you clearly informed your visitors on your privacy policy page about their different rights and that you are using Matomo?</li>
```

Comment:
What do you mean by *a* "Privacy Commissioner Office"? I can't find many (European) sources for this.
Have you asked Ronan if he has an idea on how to write this description?
And I'd write "*Have* you clearly ..."